.github/workflows/packer.yml

```
@@ -163,7 +163,7 @@ jobs:
     - name: Download base image
       if: env.release_found == 'true'
       run: |
-          oras pull quay.io/platform9/vjailbreak:base
+          oras pull docker.io/spaip9/vjailb:vjail
```

```suggestion
          oras pull docker.io/spaip9/vjailb:vjail
```
blocking merge so this is not checked in accidentally

---

removed this for now.